app/package.json

```diff
@@ -30,6 +30,7 @@
     "react-redux": "^7.2.6",
     "react-router": "^6.1.0",
     "react-router-dom": "^6.1.0",
+    "walletlink": "^2.4.7",
```

Comment:
looks like package.lock needs to be updated too,
make sure you are on a recent version of npm and install with the workspace flag

```
npm install -w app walletlink
```

Reply:
Ran that and added the lockfile, not sure if it looks reasonable though